.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,15 @@
+name: Execute Unit Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: unit-test
+        run: |
+          npm ci
+          npm run test
```

Comment:
please move the `npm run test` into `pr-check.yml` file, not a new GH action

Reply:
Yes, you're right, I think that is simpler for now.
Done here: https://github.com/StampyAI/stampy-ui/pull/193/commits/fed9bc3f2749d6f28bdf354ff0560fe003193ddd